pkg/eventing/v1beta1/client.go

```diff
@@ -42,6 +42,12 @@ type KnEventingClient interface {
 	ListTriggers() (*v1beta1.TriggerList, error)
 	// UpdateTrigger is used to update an instance of trigger
 	UpdateTrigger(trigger *v1beta1.Trigger) error
+	// CreateBroker is used to create an instance of broker
+	CreateBroker(broker *v1beta1.Broker) error
+	// DeleteBroker is used to delete an instance of broker
+	DeleteBroker(name string) error
+	// ListBroker returns list of broker CRDs
+	ListBrokers() (*v1beta1.BrokerList, error)
```

Comment:
No `GetBroker` ?

Reply:
@maximilien ? Line 48 is `DeleteBroker` ... but see my comments below, this PR is not about `kn broker describe` so no need for a "get" ...

Reply:
came in after my comment ...